ci/cloudbuild.yaml

```diff
@@ -12,10 +12,9 @@ steps:
   secretEnv:
   - 'GCP_SERVICE_ACCOUNT_KEY'
 
-- name: 'envoyproxy/envoy-build-ubuntu:41c5a05d708972d703661b702a63ef5060125c33'
-  id: 'static_analysis'
-  allowFailure: true
-  args: ['ci/static_analysis.sh']
+- name: 'envoyproxy/envoy-build-ubuntu:f94a38f62220a2b017878b790b6ea98a0f6c5f9c'
+  id: 'do_upstream_ci'
+  args: ['ci/do_ci.sh', 'dev','@envoy//test/extensions/filters/common/local_ratelimit:local_ratelimit_test', '@envoy//test/extensions/filters/http/local_ratelimit/...']
```

Comment:
could you comment here that these tests are here to verify that patch 0004-local-rate-limit....patch works successfully?

and could you also document/comment on the other tests from the other patch that were not included here, ie. whether they pass locally and/or in ci and why?

Reply:
what do you mean by other tests from other patch?

Reply:
sorry for the typo, i meant the tests listed in this comment: https://github.com/solo-io/envoy-gloo/pull/357#issuecomment-2253175286 (which i can see you already addded)